services/cloudapi/cloudapinic/nic.go

```diff
@@ -267,5 +355,21 @@ func NicSetData(d *schema.ResourceData, nic *ionoscloud.Nic) error {
 		}
 	}
 
+	if nic.Entities != nil && nic.Entities.Flowlogs != nil && nic.Entities.Flowlogs.Items != nil && len(*nic.Entities.Flowlogs.Items) > 0 {
```

Comment:
@cristiGuranIonos shouldn't we set the `flowlog` to an empty set when `len(*nic.Entities.Flowlogs.Items) == 0`? Right now it seems that we don't set anything if the set is empty.

Reply:
the entire nic will be deleted when the flowlog is empty, so it's not necessary